src/widget/form/settings/avform.cpp

```diff
@@ -321,52 +321,30 @@ void AVForm::getVideoDevices()
 
 void AVForm::getAudioInDevices()
 {
-    QString settingsInDev = Settings::getInstance().getInDev();
-    int inDevIndex = 0;
+    QStringList deviceNames;
+    deviceNames << tr("None");
+
     bodyUI->inDevCombobox->blockSignals(true);
     bodyUI->inDevCombobox->clear();
-    bodyUI->inDevCombobox->addItem(tr("None"));
-    const char* pDeviceList = Audio::inDeviceNames();
-    if (pDeviceList)
-    {
-        while (*pDeviceList)
-        {
-            int len = strlen(pDeviceList);
-            QString inDev = QString::fromUtf8(pDeviceList, len);
-            bodyUI->inDevCombobox->addItem(inDev);
-            if (settingsInDev == inDev)
-                inDevIndex = bodyUI->inDevCombobox->count()-1;
-            pDeviceList += len+1;
-        }
-    }
+    bodyUI->inDevCombobox->addItems(Audio::inDeviceNames());
```

Comment:
The `deviceNames` list should be used here and initialized with Audio::inDevNames. :) Sorry, will clean this up later today.